rocketchat-notification/main.tf

```diff
@@ -46,26 +48,42 @@ resource "aws_iam_role_policy_attachment" "lambda_xray_writeonly" {
 }
 
 
+resource "aws_iam_role_policy_attachment" "lambda_org_list_accounts" {
+  role       = aws_iam_role.security_hub_to_rocketchat_role.name
+  policy_arn = aws_iam_policy.org_policy.arn
+}
+
 resource "aws_iam_role_policy_attachment" "lambda_basic_execution_role" {
   role       = aws_iam_role.security_hub_to_rocketchat_role.name
   policy_arn = "arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole"
 }
 
+resource "null_resource" "sam_execute" {
+
+ provisioner "local-exec" {
+
+    command = "/bin/bash ./build.sh"
+  }
+}
+
 # Lambda
 resource "aws_lambda_function" "findings_to_rocketchat" {
-  filename      = "../../builds/securityhubfindings-to-rocketchat.zip"
+  depends_on = ["null_resource.sam_execute"]
+  filename      = "./builds/securityhubfindings-to-rocketchat.zip"
   function_name = "sea-send-securityhubfindings-to-rocketchat"
   role          = aws_iam_role.security_hub_to_rocketchat_role.arn
   handler       = "index.handler"
+  
 
-  source_code_hash = filebase64sha256("../../builds/securityhubfindings-to-rocketchat.zip")
```

Comment:
We'll want to keep this to trigger updates when the function code changes (with the updated path).